Formula/parquet-cli.rb

```diff
@@ -55,3 +58,20 @@ def install
     assert_match "{\"values\": \"Homebrew\"}", output
   end
 end
+__END__
```

Comment:
```suggestion
end

__END__
```
This is in the style guide (and makes it the start of the patch easier to distinguish visually).